<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendario</class>
 <widget class="QWidget" name="calendario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#salir_btn{
background:;
border:none;
	background-image: url(:/icons/icons/home.png);
}

#salir_btn:hover{
border: 1px solid rgb(238, 238, 236);
}

#agregar_btn{
background:;
border:none;
	background-image: url(:/icons/icons/pngegg(8).png);
}

#agregar_btn:hover{
border: 1px solid rgb(238, 238, 236);
}</string>
  </property>
  <widget class="QCalendarWidget" name="calendar_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_hora">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="agregar_btn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_fecha">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableCalendar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>301</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Fecha</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Evento</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="salir_btn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_evento">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
